docs/manual/kthresher.8

```diff
@@ -0,0 +1,53 @@
+.TH kthresher 8 "June 2016" "kthresher v1.2.2"
+.SH "NAME"
+kthresher - purge unused kernels
+.SH "SYNOPSIS"
+.B kthresher [-h] [-c 
+.I FILE
+.B ] [-d] [-H] [-k 
+.I [N]
+.B ] [-p] [-s] [-v] [-V]
+.SH "DESCRIPTION"
+.I kthesher
+is a utility to remove kernel packages that are marked as a candidate for autoremoval, such as those installed via unattended-upgrades or meta packages. 
```

Comment:
Please escape the "-" in the middle of words, e.g. 
```
... unattended\-upgrades ...
```

It helps to avoid line break.